Formula/gron.rb

```diff
@@ -37,6 +37,11 @@ class Gron < Formula
     :revision => "f344a1ffbe51794516e9cf2c4d58b203863d3070"
   end
 
+  go_resource "github.com/pkg/errors" do
+    url "https://github.com/pkg/errors.git",
+    :revision => "17b591df37844cde689f4d5813e5cea0927d8dd2"
```

Comment:
@ilovezfs I don't care about whose idea it is as long as everything's consistent in the end 😸 Bonus: open a PR in https://github.com/sparrc/gdm to fix `gdm brew`, and get them to tag a release.